src/main/java/com/blackducksoftware/integration/hub/alert/web/model/distribution/CommonDistributionConfigRestModel.java

```diff
@@ -24,10 +24,9 @@
 
 import java.util.List;
 
-import org.apache.commons.lang3.builder.RecursiveToStringStyle;
-import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
-
 import com.blackducksoftware.integration.hub.alert.web.model.ConfigRestModel;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
Is there a son equivalent to this?

Reply:
I wasn't able to find a good replacement. If I were to actually use Gson on the object in its toString method, I would also need to change the expected json output for all our tests. If there are empty or null values in an object that has Gson ran on it, then the values aren't added to the new json string where our tests expect value: null

Reply:
I don't want to introduce a different JSON serialization technology just to support tests.  I want to be consistent I would say go back to GSON. The tests for empty fields is handled by the entity tests where the fields are set to null.  The method: 
getEmptyEntityJson method in the test code should be in a base class and return the json string of "{}" which is an empty JSON object.